### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by phutelmyer** [target/strelka#185]
**Describe the bug**
When building the current iteration of Strelka, the following exception is thrown:

```
The command '/bin/sh -c apt-get -qq update &&     apt-get install --no-install-recommends -qq     automake     build-essential     curl     gcc     git     libtool     make     swig     python3-dev     python3-pip     python3-wheel     pkg-config     antiword     libarchive-dev     libfuzzy-dev     libmagic-dev     libssl-dev     libzbar0     python3-setuptools     redis-server     tesseract-ocr     unrar     upx     jq &&     cd /tmp/ &&     curl -OL https://exiftool.org/Image-ExifTool-12.30.tar.gz &&     tar -zxvf Image-ExifTool-12.30.tar.gz &&     cd Image-ExifTool-12.30/ &&     perl Makefile.PL &&     make &&     make install &&     cd /tmp/ &&     curl -OL https://github.com/fireeye/capa/releases/download/v$CAPA_VERSION/capa-linux &&     chmod +x /tmp/capa-linux &&     mkdir /.viv/ &&     chmod -R a+rw /.viv &&     cd /tmp/ &&     curl -OL https://s3.amazonaws.com/build-artifacts.floss.flare.fireeye.com/travis/linux/dist/floss &&     chmod +x /tmp/floss &&     cd /tmp/ &&     curl -OL https://github.com/VirusTotal/yara/archive/v$YARA_VERSION.tar.gz &&     tar -zxvf v$YARA_VERSION.tar.gz &&     cd yara-$YARA_VERSION/ &&     ./bootstrap.sh &&     ./configure --with-crypto --enable-dotnet --enable-magic &&     make && make install && make check &&     cd /tmp/ &&     curl -OL https://github.com/VirusTotal/yara-python/archive/v$YARA_PYTHON_VERSION.tar.gz &&     tar -zxvf v$YARA_PYTHON_VERSION.tar.gz &&     cd yara-python-$YARA_PYTHON_VERSION/ &&     python3 setup.py build --dynamic-linking &&     python3 setup.py install' returned a non-zero code: 2
Service 'backend' failed to build : Build failed
```

A dependency is causing issues in the build process.

**Steps to reproduce**
1. `docker-compose -f build/docker-compose.yaml up`

**Expected behavior**
Build without failure

**Screenshots**
N/A

**Release**
 - Release: 0.25.5.17 (Build on 11/28/2021)

**Additional context**

**Comment by phutelmyer** [comment]
exiftool's source is unavailable. Move references to github releases: https://github.com/exiftool/exiftool/releases/tag/12.36

**Comment by phutelmyer** [comment]
This has been fixed in #186
